src/main/java/bar/Simplest.java

```diff
@@ -5,6 +5,8 @@
  */
 public class Simplest {
 
+    public int FOO = 2;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "FOO" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S00116](https://rebloch.sonarsource.com/organizations/fab/rules#rule_key=squid%3AS00116))

[See it in SonarCloud](https://rebloch.sonarsource.com/project/issues?id=test%3Afake-project-for-tests&pullRequest=1&issues=AWHcBRsZQYL-lH1H_cI9&open=AWHcBRsZQYL-lH1H_cI9)

[ISSUE_KEY]: AWHcBRsZQYL-lH1H_cI9